platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/ConfigurationAdminImpl.java

```diff
@@ -294,7 +294,11 @@ private void addConfigurationData(Service service, Configuration[] configs) {
                     .filter(metatype -> AttributeDefinition.PASSWORD == metatype.getType())
                     .forEach(metatype -> {
                         String passwordProperty = metatype.getId();
-                        propertiesTable.put(passwordProperty, "password");
+                        if (propertiesTable.get(passwordProperty) == null){
```

Comment:
✏️ My strong preference is to separate the form/structure of streaming code from the internal complexity. I'm not opposed to one-liner lambdas, but anything more than that makes the stream hard to read. I'd extract the guts of this out to a method.